mtd-utils/build.sh

```diff
@@ -11,7 +11,7 @@ if [ "$(uname)" = "Darwin" ]; then
 	mkdir -p "$PREFIX_MTD_UTILS_BUILD"
 
 	pushd "$PREFIX_MTD_UTILS_BUILD"
-	if [ -n "$CLEAN" ]; then
+	if [ "$B_CLEAN" = "y" ]; then
```

Comment:
this should not depend on `B_CLEAN` variable but on existence of target files

Reply:
Seems to be a better solution - updated. I'd merge it as it is for now.